### ISSUE TITLE
Multiline label

### ISSUE THREAD

**Issue by hons82** [uias/Tabman#304]
### New Issue Checklist

- [X] Updated Tabman to the latest version
- [X] I took a look at the [Troubleshooting Guide](https://github.com/uias/Tabman/blob/master/Docs/TROUBLESHOOTING.md)
- [X] I searched for [existing GitHub issues](https://github.com/uias/Tabman/issues)

### Issue Description

That would be a possible improvement in case...
I'd need to setup a tab with 2 lines of text, which is currently impossible as far as I've seen.

### Other useful things

I've tried to do a quick modification to the code by adding those two lines
```
button.titleLabel?.numberOfLines = 0
button.titleLabel?.textAlignment = .center
```
to the `addBarButtons` function in the `TabmanButtonBar` class. Now I can put two labels like these 
```
"tab_1" = "I'm a one-liner";
"tab_2" = "I need a bit more space\nso I'm going to occupy two lines\nor more...";

```
as header to obtain the result I need.

Now there are some questions

1. Does this make sense to add as a feature?
2. Does it interfere with other features that I haven't seen so far?
3. If I'd create a PR with this should it be customisable?

**Comment by msaps** [comment]
@hons82 the flexibility would certainly be nice - but I'm currently in the middle of building Tabman 2.0 which will make this sort of flexibility much easier to implement and available 😄. So it's probably not worth a PR currently as I'm only really planning on support 1.x with fixes until 2.0 is available. 

🎉

**Comment by msaps** [comment]
@hons82 just released [Tabman 2](https://github.com/uias/Tabman/releases/tag/2.0.0) which makes creating custom bar buttons much easier. Check out the docs [here](https://uias.github.io/Tabman/master/going-custom.html#custom-bar-buttons)
